Gopistol/demo/src/main/java/com/example/demo/comment/domain/Comment.java

```diff
@@ -0,0 +1,38 @@
+package com.example.demo.comment.domain;
+
+import com.example.demo.member.domain.Member;
+import com.example.demo.post.domain.Post;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Data
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor
+public class Comment {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  private String comment;
+
+  @ManyToOne(fetch = FetchType.LAZY)
+  private Member member;
+
+  @ManyToOne(fetch = FetchType.LAZY)
+  private Post post;
+
+  public static Comment create(Member member, Post post, String comment) {
+    return new Comment(null, comment, member, post);
+  }
```

Comment:
```suggestion
  public static Comment of(Member member, Post post, String comment) {
    return new Comment(null, comment, member, post);
  }
```
[정적 팩토리 메서드 네이밍 컨벤션](https://velog.io/@saint6839/%EC%A0%95%EC%A0%81-%ED%8C%A9%ED%86%A0%EB%A6%AC-%EB%A9%94%EC%84%9C%EB%93%9C-%EB%84%A4%EC%9D%B4%EB%B0%8D-%EB%B0%A9%EC%8B%9D) 참고하기!